src/main/java/com/softserve/repository/impl/TemporaryScheduleRepositoryImpl.java

```diff
@@ -42,9 +48,9 @@ public Long isExistTemporaryScheduleByVacationByDateWithIgnoreId(Long id, LocalD
                         "where  t.date = :date and t.vacation = :vacation and t.semester.id = :semesterId and t.id!= :id and t.period = null " +
                         "and t.teacher = null  and t.scheduleId = null ")
                 .setParameter("date", date)
```

Comment:
why not creating a constant for this literal?

Reply:
Okay I create